docs/api/api/directory.xsd

```diff
@@ -36,6 +36,7 @@
       <xs:attribute name="name" type="xs:string"/>
       <xs:attribute name="size" type="xs:string"/>
       <xs:attribute name="md5" type="xs:string"/>
+      <xs:attribute name="hash" type="xs:string"/>
```

Comment:
This is probably out of scope here, but I'd suggest to call this parameter `sha256hash` for clarity.

Reply:
yeah, unfortunatly this would break the api ... we could just introduce this in addition.

Reply:
No, the value is prefixed with the hash name, e.g. `sha256:abcdef...`. That's why it's just named `hash`.

Reply:
Oh! That definitely belongs into the the documentation of that attribute then, because I would've just dumped the sha256 hash there.

Reply:
right, added